cocos/core/geometry/triangle.ts

```diff
@@ -176,7 +198,7 @@ export class Triangle {
         return this._type;
     }
 
-    protected readonly _type: number;
+    protected readonly _type: number = enums.SHAPE_TRIANGLE;
```

Comment:
Why not use declare?

Reply:
Only use `declare` when properties are initialized in `constructor`. Otherwise, it doesn't need to use `declare` keyword.

Reply:
It is initialized in constructor.